Policies/Kubernetes/host-network-ports/azurepolicy.json

```diff
@@ -0,0 +1,80 @@
+{
+  "properties": {
+    "displayName": "Control whether a Pod may use the Host Network, and whitelist Host Ports in a Kubernetes Cluster",
```

Comment:
word 'whitelist' should not be used in policies that will be published as built-ins.